tests/integration/address/test_models.py

```diff
@@ -169,6 +179,12 @@ def test_populated_shipping_address_has_same_summary_user_address(self):
         a.populate_alternative_model(sa)
         self.assertEqual(sa.summary, a.summary)
 
+    def test_summary_value(self):
```

Comment:
This seems to be testing essentially the same thing as the test below it?

Reply:
Indeed.

Reply:
> I think the hash has to be consistent given what we use it for. Maybe we need a migration that converts the old Python 2 hashes into the new ones.

Maybe. In any case, user is free to skip data migration - it does not modify database schema obviously.

Reply:
> This seems to be testing essentially the same thing as the test below it?

Yes, but tests validates summary generation from different set of fields, so I think we should keep them both.